Katan/db/schema.rb

```diff
@@ -11,3 +11,69 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+ActiveRecord::Schema.define(version: 20160902123105) do
```

Comment:
__[rubocop]__
* Separate every 3 digits in the integer portion of a number with underscores(_). [:ref](https://github.com/bbatsov/ruby-style-guide#underscores-in-numerics)

<!-- Comment by [SideCI](https://sideci.com) -->